icekit/plugins/image/abstract_models.py

```diff
@@ -144,10 +202,22 @@ def title(self):
         """
         self.title_override = ''
 
+    @property
+    def credit(self):
```

Comment:
This function is buggy (my bad). Needs to return self.image.credit at all times. Also fix up the docstring.